flow-api/src/main/java/com/sequenceiq/flow/api/FlowEndpoint.java

```diff
@@ -80,4 +86,23 @@ public interface FlowEndpoint {
     @ApiOperation(value = "Check if there is a running flow for flow id", produces = "application/json", notes = "Flow log operations",
             nickname = "hasFlowRunningByFlowId")
     FlowCheckResponse hasFlowRunningByFlowId(@PathParam("flowId") String flowId);
+
+    @GET
+    @Path("/check/flowIds")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Get flow logs by a list of flow ids - Input size max 50", produces = "application/json", notes = "Flow log operations",
+            nickname = "getFlowLogStatusesByFlowIds")
+    List<FlowLogResponse> getFlowLogStatusesByFlowIds(@NotNull @Size(max = 50, message = "Input size should not be over 50")
+        @QueryParam("flowIds") List<String> flowIds,
+        @Min(value = 1, message = "Page size should be greater than 0") @QueryParam("size") int size, @QueryParam("page") int page);
```

Comment:
the path indicates it is an API about flow check but the response object is `FlowLogResponse`. based on existing API, path should be `/logs/flowIds` or response object should be `List<FlowCheckResponse>`

Reply:
done please verify